hack/install.sh

```diff
@@ -194,7 +161,8 @@ EOF
 }
 
 install_faas_cli() {
-  curl -sLS https://cli.openfaas.com | $SUDO sh
+  arkade get faas-cli
+  $SUDO mv $HOME/.arkade/bin/faas-cli /usr/local/bin/
```

Comment:
I think you will also need to [chmod](https://linuxhandbook.com/chmod-command/) every binary so that the world can read and execute.

```
chmod o+rx /usr/local/bin/faas-cli

# or

chmod 755 /usr/local/bin/faas-cli
```

The [`install` command](https://www.geeksforgeeks.org/install-command-in-linux-with-examples/) can also do this in one line:

```
$SUDO install -m 755 $HOME/.arkade/bin/faas-cli /usr/local/bin/
```

Reply:
Done